projectJava/UniversityGroup/src/MyClass.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+public class MyClass {
+        public static void main(String[] args){
+                UniversityGroup group = new UniversityGroup("Math123", 2009);
+                String[] students = new String[] {"std1","std2","std3","std4","std5","std6"};
+                UniversityGroup groupSecond = new UniversityGroup("Math321", 2007, students);
+                group.addStudents(students);
+                groupSecond.addStudents(students);
```

Comment:
Please add another students array to simulate situation with adding students to the existing students